pom.xml

```diff
@@ -179,6 +180,13 @@
                 <artifactId>hamcrest-all</artifactId>
                 <version>1.3</version>
             </dependency>
+
+            <!-- this should only be included in the uber jar for standalone execution or as a test dep -->
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-classic</artifactId>
+                <version>1.2.3</version>
+            </dependency>
```

Comment:
we include this in the harry-integration-external uber jar for standalone execution